Doc/using/configure.rst

```diff
@@ -851,6 +851,13 @@ Libraries options
 
    .. seealso:: :option:`LIBMPDEC_CFLAGS` and :option:`LIBMPDEC_LIBS`.
 
+.. option:: --with-libmpdec
+
+   Whether to build the ``_decimal`` extension module (default is yes).
```

Comment:
You need to update the removal versioning for vendored sources above (it's currently indicating 3.15 but we decided to defer it right?)